src/cmd/add.nim

```diff
@@ -81,11 +82,31 @@ proc paxAdd*(input: string, strategy: string): void =
   echoMod(cfMod, moreInfo = true)
   echo ""
 
-  returnIfNot promptYN("Are you sure you want to install this mod?", default = true)
+  returnIfNot promptYN("Are you sure you want to install this mod?",
+      default = true)
 
   echoInfo "Installing ", cfMod.name.cyanFg, ".."
   manifest.installMod(cfMod.projectId, cfModFile.fileId)
 
+  echoInfo "Resolving Dependencies..."
+  ## Get Dependendencies
+  if len(cfModFile.dependencies) > 0:
+    if promptYN("Do you want to install dependencies?", default = true):
+      for id in cfModFile.dependencies:
+        let cfMod = waitFor(fetchMod(id))
+        if manifest.isInstalled(id):
+          echoInfo cfMod.name.cyanFg, " is already installed, skipping"
+          continue
+        let cfModFiles = waitFor(fetchModFiles(id))
+        let selectedCfModFile = cfModFiles.selectModFile(manifest, strategy)
+        if selectedCfModFile.isNone:
+          echoError "Warning: unable to resolve dependencies."
+        let cfModFile = selectedCfModFile.get()
+        echoInfo "Installing ", cfMod.name.cyanFg, ".."
+        manifest.installMod(id, cfModFile.fileId)
+  else:
+    echoInfo "No dependencies found..."
```

Comment:
I don't think we need an extra message if there a no dependencies. You could remove that.